netci.groovy

```diff
@@ -638,6 +638,10 @@ combinedScenarios.each { scenario ->
                                         if (Constants.jitStressModeScenarios.containsKey(scenario)) {
                                             if (enableCorefxTesting) {
                                                 // Sync to corefx repo
+                                                // Move coreclr files to a subdirectory, %workspace%/clr. Otherwise, corefx build 
+                                                // thinks that %workspace% is the project base directory.
+                                                buildCommands += "powershell new-item clr -type directory -force"
+                                                buildCommands += "powershell foreach (\$x in get-childitem -force) { if (\$x.name -ne \'clr\') { move-item \$x clr }}"
```

Comment:
Thanks for review.  I didn't know about -Command... However, the powershell commands above work well :) I tested them locally and I also confirmed it from http://dotnet-ci.cloudapp.net/job/dotnet_coreclr/job/x64_chk_winnt_fx0_prtest/14/console.